src/rules/noUnusedVariableRule.ts

```diff
@@ -88,7 +133,26 @@ class NoUnusedVariablesWalker extends Lint.RuleWalker {
     }
 
     public visitNamespaceImport(node: ts.NamespaceImport) {
-        this.validateReferencesForVariable(node.name.text, node.name.getStart());
+        let importDecl = <ts.ImportDeclaration>node.parent.parent;
+        let modSpec = importDecl.moduleSpecifier.getText();
+
+        // extract the unquoted module being imported
+        let match = modSpec.match(MODULE_SPECIFIER_MATCH);
+
+        let isReactImport = match && REACT_MODULES.indexOf(match[1]) !== -1;
+        if (this.hasOption(OPTION_REACT) &&
+            isReactImport &&
```

Comment:
we normally put the && operators on the next line instead of trailing the previous line